src/feature-selector-form/feature-selector-form.tsx

```diff
@@ -15,6 +26,7 @@ import {
 import styles from './feature-selector-form.module.css';
 import { FeatureSelectorTree } from './sub-components';
 import { SelectedFeatureDetails } from './sub-components/selected-feature-details';
+import { getRelevantTaskIds } from '../combined-selectors/relevant-task-ids';
```

Comment:
I needed to expose this because the `selectRelevantTaskIds` selector is coming from the global state and will only update when the issue details or reporting config in the global state change.

In this component, we only want to calculate the relevant tasks based on the local state of the `selectedFeatureId`, which is not reflected in the global state yet.

That's why I exposed this and decided to use it with the local state values in this component.